### DISCUSSION TITLE
Failed to Load Extension Package When Calling CRT Extension from POS

### DISCUSSION THREAD

**Discussion by tatjotta** [microsoft/Dynamics365Commerce.Solutions#282] [Q&A]
I'm implementing a process to call the CRT extension from the POS, but the loading of the extension package fails. It seems that the DataService being generated is referencing the wrong location. Is there a way to change some settings to correct this reference destination?

During package loading, the StoreCommerceApp tries to reference local.storecommerce.dynamics.com, but the ’DataService’ is actually located at extensions.local.storecommerce.dynamics.com

<img width="385" alt="image" src="https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/118195096/c788015f-d292-447f-9963-d98e7facc344">

<img width="211" alt="localstorecommerce" src="https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/118195096/50140633-6688-48c7-9bb4-0bc49341f6b5">

<img width="308" alt="extensionlocalstorecommece" src="https://github.com/microsoft/Dynamics365Commerce.Solutions/assets/118195096/f9d9b303-e6b8-46e6-a108-707b03f0457e">

**Comment by zhangguanghuib** [comment]
Please follow the bottom part of this document to fix this issue,   this doc mentioned the issue of failing to load extension package issue

https://github.com/zhangguanghuib/NewCommerceSDK/blob/main/KnowledgeArticles-Development/0.1%20Commerce%20SDK%20Development%20Environment%20Preparation.md

**Comment by tatjotta** [comment]
Thank you for your response. Unfortunately, it does not seem to address my specific question. I am interested in understanding the errors that appear in the Console tab, as mentioned at the bottom of the document.

I am attempting to fetch data from DataService/DataServiceRequests.g, but it appears that this file is located within the extensions. Could you help clarify why there is an attempt to fetch from https://local.storecommerce.dynamics.com?

Additionally, I have been unable to locate where the generation of DataService/DataServiceRequests.g is defined. Any guidance on this would be greatly appreciated.
